specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2021-12-01/bms.json

```diff
@@ -5485,6 +5500,13 @@
           "$ref": "#/definitions/RetentionPolicy",
           "description": "Retention policy with the details on backup copy retention ranges."
         },
+        "tieringPolicy": {
+          "description": "Tiering policy to automatically move RPs to another tier\r\nKey is Target Tier, defined in RecoveryPointTierType enum.\r\nTiering policy specifies the criteria to move RP to the target tier.",
```

Comment:
>Key is Target Tier [](http://example.com/codeflow?start=86&length=18)

If a target tier is not in the dictionary, what does it default to?

Reply:
In a GET call, all available tiers will be included by the service. In a CreateOrUpdate call, any target tiers omitted by user will result in service retaining the existing policy for that target tier.

Reply:
Please call out the default behavior in description

Reply:
Added